### PR TITLE
PR - Updated Toast Doc

### LINKED ISSUE #180: Document Toast component
PR: https://github.com/webforj/webforj/pull/648

# Toasts

1. **What is a Toast Notification?**

2. **Basic Syntax Using the "show" Method**
   - Provide a demonstration of how to use the "show" method to display a toast notification.

3. **Durations: Configuring the Timeout of the Toast**
   - Include working code samples to show how to set the duration of the toast.
   - Explain how to create persistent toasts that do not disappear automatically.

4. **Placement**
   - Document where the toast can be positioned on the screen.
   - Provide a demonstration of different positioning options.

5. **Stacking**
   - Explain that multiple notifications can be displayed simultaneously, stacking vertically.
   - Describe how the ordering of stacked notifications depends on their placement.

6. **Actions and Interactivity**
   - Demonstrate how to add interactive content to notifications, allowing users to perform actions directly from the toast.

7. **Themes (Under Styling)**
   - Show different styling options for toasts, including custom themes.
   - Provide a demonstration of how to apply and customize these themes.

### PR DESCRIPTION
**Pull request by laurenic0l**
Updated the Toast doc, links, etc.

closes #180

### CODE REVIEW

**Review by MatthewHawkins — DISMISSED** [review]

File: docs/components/toast.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+## Duration
+
+You can configure `Toast` notifications to disappear after a set duration or persist on the screen until they're dismissed, depending on your needs. By default, a `Toast` closes automatically after 3000 milliseconds. You can customize the duration with the `setDuration()` method, or simply supply a duration parameter to the constructor or the `show()` method.
+
+:::info Default Duration
+By default, the `Toast` will have a duration of 3000 milliseconds.
+:::
+
+```java
```
Comment (hyyan) [inline docs/components/toast.md:62]: We should include a tip or note explaining that, unlike other components, a toast doesn't need to be added to a container. The `open` method will automatically attach the toast directly to the first app frame.

**Comment by MatthewHawkins** [comment]
@hyyan Tested with the updated demo URLs, everything here looks good to me, ready for your review.

File: docs/components/toast.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+<ComponentDemo 
+path='https://demo.webforj.com/webapp/controlsamples/toastplacementdemo?'
+javaE='https://raw.githubusercontent.com/webforj/webforj-docs-samples/refs/heads/main/src/main/java/com/webforj/samples/views/toast/ToastPlacementDemoView.java'
+cssURL='https://raw.githubusercontent.com/webforj/ControlSamples/main/src/main/resources/css/toaststyles/toastplacementdemo.css'
+height='300px'
+/>
+
+By customizing the placement of your `Toast` notifications, you can ensure that users receive information in a way that's appropriate for any given app, screen layout, and context.
+
+### Stacking
```
Comment (hyyan) [inline docs/components/toast.md:111]: Should be `## Stacking`. Header level 2

File: docs/components/toast.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+toast.open();
+```
+
+<ComponentDemo 
+path='https://demo.webforj.com/webapp/controlsamples/toastdemo?'
+javaE='https://raw.githubusercontent.com/webforj/webforj-docs-samples/refs/heads/main/src/main/java/com/webforj/samples/views/toast/ToastDemoView.java'
+height='200px'
+/>
+
+## Usages
```
Comment (hyyan) [inline docs/components/toast.md:37]: This section feels repetitive compared to the introduction. Let's merge the "Basics" and "Usages" sections, and use the sample from the "Basics" section here. The cookies demo can then be included in the "Actions and Interactivity" section.

File: docs/components/toast.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+:::
+
+<ComponentDemo 
+path='https://demo.webforj.com/webapp/controlsamples/toastthemedemo?'  
+javaE='https://raw.githubusercontent.com/webforj/webforj-docs-samples/refs/heads/main/src/main/java/com/webforj/samples/views/toast/ToastThemeDemoView.java'
+cssURL='https://raw.githubusercontent.com/webforj/ControlSamples/main/src/main/resources/css/toaststyles/toastthemedemo.css'
+height='200px'
+/>
+
+### Shadow parts
```
Comment (hyyan) [inline docs/components/toast.md:158]: The tables are not loading, and some are missing. Please check with other documents for reference.

**Review by hyyan — CHANGES_REQUESTED** [review]

**Comment by hyyan** [comment]
@laurenic0l looks good overall. I requested few changes

File: docs/components/toast.md (hunk 3 above)
Comment (hyyan) [reply docs/components/toast.md:37]: @laurenic0l, maybe wire the buttons to simply close the toast. ?

**Review by hyyan — DISMISSED** [review]

**Review by MatthewHawkins — APPROVED** [review]

**Pull request merged**